nntrainer/models/neuralnet.cpp

```diff
@@ -877,6 +878,10 @@ int NeuralNetwork::setDataBuffer(const DatasetModeType &mode,
 
 int NeuralNetwork::getLayer(const char *name,
                             std::shared_ptr<ml::train::Layer> *layer) {
+  if (compiled) {
+    return ML_ERROR_NOT_SUPPORTED;
+  }
```

Comment:
```compiled```? ```!compiled```?

Reply:
It's compiled.
Once the model is compiled then the graph structure is fixed.
So attempts to modifying the graph structure eg) changing input shape or input layer is prohibithed if model is compiled